docs/configuration.mdx

```diff
@@ -44,32 +44,34 @@ Usage: `/config get config:Log channel` (to view) or `/config logs channel:<chan
 You can either view or set the channel where suggestion results will be posted after a suggestion is approved or denied. The options to pass through follow the same as setting the [Suggestions Channel](#suggestions-channel).
 
 ## DM Responses
-Usage: `/config get config:Dm responses` or `/config dm <enable|disable>`
+Usage: `/config get config:Dm responses` (to view) or `/config dm <enable|disable>` (to set)
 
 You can either view if DM responses are enabled/disabled or enable/disable them if you wish.
 
 When disabling DM responses, the bot won't DM the user on these actions:
 
-- A suggestion is submitted
+- A suggestion is submitted (an ephemeral message will be sent instead)
 - A suggestion is approved
 - A suggestion is rejected
-- A note is added to a suggestion
 
-Users are also able to configure their own DM settings using the `/dm <enable|disable>` command. This allows a user to choose their own preferences. However, they cannot override your server's settings.
+Users are also able to configure their own DM settings using the `/dm <enable|disable>` command. This allows a user to choose their own preferences at the guild level. However, they cannot override your server's settings.
```

Comment:
```suggestion
Users are also able to configure their own DM settings using the `/dm <enable|disable>` command. This allows a user to choose their own preferences. However, they cannot override your server's settings.
```

User configurations take effect globally, not per guild. Or at-least it should be globally